source/support/spawn_vehicle.sqf

```diff
@@ -21,11 +21,9 @@ if (commandpointsblu1 >= _cost) then {
 	hint "Vehicle ready! check map for location";
 	commandpointsblu1 = commandpointsblu1 - _cost;
 	ctrlSetText [1000, format["%1",commandpointsblu1]];
-	_vehic = _name createVehicle [0,0,0];

```

Comment:
if the change really fixes the issue, it'll be ok to have it, we don't spawn dozens of vehicles anyways